tests/indexes_/models.py

```diff
@@ -10,6 +10,7 @@ class Data(EmbeddedModel):
 
 class Article(models.Model):
     headline = models.CharField(max_length=100)
+    object_id = ObjectIdField()
```

Comment:
I think it would be cleaner to create a separate model (`SearchIndexTestModel`?) with all the fields needed for testing SearchIndex data types rather than adding a bunch of non-semantic field to `Article`. As is, it doesn't seem like we have complete test coverage for `search_index_data_types()`.